src/cli.js

```diff
@@ -60,11 +61,32 @@ function noIssue(result) {
   );
 }
 
-function prettify(caption, deps, oneline) {
+function prettify(
+  caption,
+  deps,
+  oneline,
+  rootDir = null,
+  printJsonLineInKey = null,
+) {
   if (oneline) {
     return deps.length ? [caption, deps.join(' ')] : [];
   }
-  const list = deps.map((dep) => `* ${dep}`);
+
+  const packageJson =
+    rootDir && printJsonLineInKey
+      ? fs.readFileSync(path.join(rootDir, 'package.json'), 'utf8')
+      : null;
```

Comment:
Open the package.json if needed (only on default output)

May be optimised to avoid memory consumption with a lot of unused deps (but the purpose of depcheck is to have a few unused deps, so extra optimization may be overkill ?)